tests/test_db.py

```diff
@@ -756,23 +816,73 @@ def test_source_empty(self):
         with self.assertRaisesRegex(ValueError, SOURCE_EMPTY_ERROR):
             db.add_identity(uidentity, '1234567890ABCDFE', '')
 
+    def test_source_whitespaces(self):
+        """Check whether an identity with a source composed by whitespaces cannot be added"""
+
+        uidentity = UniqueIdentity.objects.create(uuid='1234567890ABCDFE')
+
+        with self.assertRaisesRegex(ValueError, SOURCE_WHITESPACES_ERROR):
+            db.add_identity(uidentity, '1234567890ABCDFE', '  ')
+
+        with self.assertRaisesRegex(ValueError, SOURCE_WHITESPACES_ERROR):
+            db.add_identity(uidentity, '1234567890ABCDFE', '\t')
+
+        with self.assertRaisesRegex(ValueError, SOURCE_WHITESPACES_ERROR):
+            db.add_identity(uidentity, '1234567890ABCDFE', ' \t ')
+
     def test_data_none_or_empty(self):
-        """Check whether new identities cannot be added when identity data is None or empty"""
+        """Check whether new identities cannot be added when identity data is None, empty or composed by whitespaces"""
```

Comment:
This comment is too long. Please split it in two lines.

Reply:
Ok, done. I have changed it following this format:
```python
def my_test(self):
"""
<line1>
<line2>
"""
code
```

Is this what you expect? Thanks in advance!